### DISCUSSION TITLE
Mimir to mimir

### DISCUSSION THREAD

**Discussion by rdharani19** [grafana/mimir#2316] [General]
Is it possible to pull  metrics from one mimir cluster and push to an other ? 

( or mirror - i checked upon mirroring using envoy, but it is not suiting my need, I am looking for something like 
metrics-> stage-1 mimir ->stage-2 mimir) 
In my current setup i have Prometheus HA pairs in place of the stage-1 mimir, but was looking if it is even an possibility to use mimir in place of the Prometheus HA cluster

**Comment by pracucci** [comment]
There's no direct way to relay metrics from a Mimir cluster to another Mimir cluster. To help you better I would need to learn more about your actual use case, to check if you could achieve the same in a different way.

For example, what does promote a sample to move from "stage-1 mimir" to "stage-2 mimir"? If metrics need to be written at the same time to both Mimir clusters, then a common solution is to configure 2 remote write URLs in your Prometheus HA pairs, having them writing to both Mimir clusters.

**Reply by rdharani19** [reply]
Thanks Marco,  

We wanted the stage-1 mimir, to be more like a remote collection clusters( sort of like small clusters in several remote locations, which forward the collected metrics to mimir-2)  and mimir-2 is the main cluster that most of the infra components would write metrics to(along with these collected metrics from the remote locations) Also, Mimir-2 is the datastore where the end users and Grafana would mostly query to.

**Reply by pracucci** [reply]
I think what you want to do is achievable just setting 2 remote write endpoints in your Prometheus. You can also have relabelling rules per remote write endpoint, to drop series you don't want in the stage-1 or stage-2 Mimir cluster.
